examples/coverage/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
```

Comment:
This crate contains a lot of commented code: shall we clean this up?
Also, can you add a README here to explain what this example is about?

Reply:
Isn't this redundant with `tests/`?

Reply:
Possibly, this could be moved into some of the tests. We will probably want some tests for the semantics, and not just testing cases of code generation.